<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPane</class>
 <widget class="QWidget" name="ControlPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Pane</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Index of displayed frame:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLCDNumber" name="frame_index_widget">
     <property name="digitCount">
      <number>9</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="start_stop_button">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pause_resume_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Write output:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="write_output_checkbox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="status_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
